.github/CONTRIBUTING.md

```diff
@@ -219,3 +219,24 @@ for (var/client/C in world)
 All tables for the database should be prefixed according to the following list:
 * `ss13_` for tables in which ingame data is held.
 * `discord_` for tables in which BOREALIS data is held.
+
+# HTML UI Standards
+
+### UI conversion policy
+Due to our current situation with 5 different HTML UI systems we are now enforcing new policy that all new uis without exception should be made in VueUi ui system. This policy also applies to editing existing uis, but there is few exceptions:
+
+ 1. Modification is security / severe bug fix.
+ 0. It is typo fix.
+ 0. Touched UI file is too large.
+ 0. VueUi can't accommodate that type of UI.
+
+### Responsiveness
```

Comment:
whaddoes this mean?

Reply:
It means that uis should not require you to click refresh.